scripts/Testing/Testcases/NCR_build.csh

```diff
@@ -85,7 +85,7 @@ set NTASKS_CPL  = `./xmlquery NTASKS_CPL  --value`
 ./case.setup --clean --test-mode
 ./case.setup
 
-./case.clean_build
+./case.build --clean
```

Comment:
I have rebased to the head of Jim's branch - so these are going away. I will then rebase to master before issuing a new pull request.